pkg/common/spark.go

```diff
@@ -301,12 +304,18 @@ const (
 	// LabelSparkAppName is the name of the label for the SparkApplication object name.
 	LabelSparkAppName = LabelAnnotationPrefix + "app-name"
 
+	// LabelSparkConnName is the name of the label for the SparkConnect object name.
+	LabelSparkConnName = LabelAnnotationPrefix + "conn-name"
```

Comment:
```suggestion
	LabelSparkConnectName = LabelAnnotationPrefix + "connect-name"
```

I think it's ok to use full name, "Connect" , for clarify.

Reply:
Updated.